SC_redesign/Crop_and_Soil/soil/soil_data.py

```diff
@@ -101,3 +130,18 @@ def soil_water_factor(self):
     #   - soil evaporation adjusted is always less than or equal to maximum soil evaporation
     #   - potential evapotranspiration adjusted is always less than or equal to maximum evapotranspiration
     #   - average slope fraction is always in the range 0 to 1 inclusive (I think)
+
+    @property
+    def profile_bulk_density(self):
+        """
+
+        Returns: average bulk density of the soil profile based on the bulk density of each soil layer and weighted by
+            the thickness of that layer
+
+        """
```

Comment:
You can document `@properties` as though they are regular attributes if it makes sense to do so.

```suggestion
        """average bulk density of the soil profile based on the bulk density of each soil layer, weighted by
            the thickness
        """
```

Reply:
Yes that would be better. I was just going along with the docstrings that were being autogenerated for me